tools/vipps-login-api.postman_environment.json

```diff
@@ -1,47 +1,40 @@
 {
-	"id": "b8901041-7378-49d4-8cf4-60a0ba3b923c",
+	"id": "dea2b12b-55d9-4c7a-aa6e-52d59575c313",
 	"name": "vipps-login-api",
 	"values": [
 		{
 			"key": "well-known_uri",
 			"value": "\"Add url to well-known endpoint for used environment\"",
-			"description": "Url to well-known endpoint for th OIDC service",
 			"enabled": true
 		},
 		{
 			"key": "client_id",
 			"value": "\"Add received client id\"",
-			"description": "Client id for merchant",
 			"enabled": true
 		},
 		{
 			"key": "client_secret",
 			"value": "\"Add received client secret\"",
-			"description": "Client secret for merchant",
 			"enabled": true
 		},
 		{
 			"key": "redirect_uri",
 			"value": "\"Add redirect uri\"",
-			"description": "Redirect uri to merchant",
 			"enabled": true
 		},
 		{
 			"key": "response_type",
 			"value": "code",
-			"description": "",
 			"enabled": true
 		},
 		{
 			"key": "scope",
 			"value": "openid",
-			"description": "Requested scopes",
 			"enabled": true
 		},
 		{
 			"key": "code",
 			"value": "\"Add code from url\"",
-			"description": "Code used to fetch access token after finishing a Vipps login",
 			"enabled": true
```

Comment:
Why remove description?

Reply:
I didn't notice that it removed those! I didn't do it  myself.

Reply:
I have added them back manually. Maybe the field is no longer present in a newer version of postman.

Reply:
I added them back manually.